src/cashweb/wallet/index.ts

```diff
@@ -418,9 +418,10 @@ export class Wallet {
     console.log('Broadcasting forwarding txn', transaction)
     const txHex = transaction.toString()
     try {
-      const electrumClient = await this.electrumClientPromise
-      assert(electrumClient, 'missing client in forwardUTXOsToAddress')
-      await electrumClient.request('blockchain.transaction.broadcast', txHex)
+      const chronikClient = this.chronikClient
```

Comment:
Is an await no longer required here? Sometimes the electrumClient would disconnect and this would attempt to reconnect if it had.

Reply:
This uses the REST API, which is independent from the WebSocket. Should it wait for the WS to connect before attempting to broadcast?

Reply:
I see. Makes sense.